barebones/core_portme.c

```diff
@@ -33,15 +34,20 @@ Original Author: Shay Gal-on
 	volatile ee_s32 seed2_volatile=0x8;
 	volatile ee_s32 seed3_volatile=0x8;
 #endif
-	volatile ee_s32 seed4_volatile=ITERATIONS;
+	volatile ee_s32 seed4_volatile=2; //ITERATIONS;
 	volatile ee_s32 seed5_volatile=0;
 /* Porting : Timing functions
 	How to capture time and convert to seconds must be ported to whatever is supported by the platform.
 	e.g. Read value from on board RTC, read value from cpu clock cycles performance counter etc. 
 	Sample implementation for standard time.h and windows.h definitions included.
 */
 CORETIMETYPE barebones_clock() {
-	#error "You must implement a method to measure time in barebones_clock()! This function should return current time.\n"
+//	#error "You must implement a method to measure time in barebones_clock()! This function should return current time.\n"
+	//based off of bsg_manycore implimentation since we need a timer	
+        static CORETIMETYPE fake_timer=0;
+        fake_timer += 100;
+        return fake_timer;
```

Comment:
We do actually have mtime CSR, if you want to try that out. It hasn't been tested well, so YMMV.  Maybe code in that inline assembly, but comment it out for now?

Reply:
I'm not sure what you mean? I was following how BSG_manycore handled this part since other functions call this in the code so would be less overall editing but I could remove all the time parts for now if that's what you're saying

Reply:
Leave it as is, but put a comment
// BlackParrot support mtime, so we could possibly use that for actually accurate timing

Reply:
You got it